credentialproviderpackage/charts/credential-provider-package/templates/daemonset.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "credential-provider.fullname" . }}
+  namespace: {{ .Release.Namespace | default .Values.defaultNamespace | quote }}
```

Comment:
Is this going to be used in workload clusters? I don't think we want .Release.Namespace here.

Reply:
Yes this can be used on workload clusters. Doesn't the targetNamepsace need the {{ .Release.Namespace }} to be able to install to seperate namespace?

Reply:
The resources should go to the target namespace which may or may not be the same as the release namespace.